src/main/java/com/terraforged/mod/server/command/TerraCommand.java

```diff
@@ -274,11 +272,11 @@ private static int findTerrainAndBiome(CommandContext<CommandSource> context) th
         Search terrainSearch = new TerrainSearchTask(pos, target, getChunkGenerator(context), generator);
         Search search = new BothSearchTask(pos, biomeSearch, terrainSearch);
         int identifier = doSearch(server, playerID, search);
-        context.getSource().sendFeedback(createIdentifierMessage(identifier)
+        context.getSource().sendFeedback(createPrefix(identifier)
                                          .appendSibling(new StringTextComponent(" Searching for "))
-                                         .appendSibling(createBiomeDisplayMessage(biome))
+                                         .appendSibling(createTitleWithHover(biome.getDisplayName().getFormattedText(), biome.getRegistryName().toString()))
```

Comment:
`biome.getDisplayName()` is client-only so cannot be used. Need to revert to just displaying the registry name